ansible/roles/nginx/files/s142.okserver.org/icinga

```diff
@@ -0,0 +1,23 @@
+server {
+    server_name icinga.okserver.org;
+    index /wiki/index.php;
+    root /var/www/html;
+
+    access_log /var/log/nginx/icinga.access.log;
+    error_log /var/log/nginx/icinga.error.log;
+    location ~ ^/icingaweb2/index\.php(.*)$ {
+      fastcgi_pass 127.0.0.1:9000;
```

Comment:
As of now, fpm is configured to listen on the unix port, so the right config line here is `unix:/var/run/php5-fpm.sock`